src/responsive-layout.tsx

```diff
@@ -73,4 +77,29 @@ const ResponsiveLayout: React.FC<Props> = ({children, ...props}) => (
     </InternalResponsiveLayout>
 );
 
+export const ResetResponsiveLayout: React.FC<{
+    children: React.ReactNode;
+    mobile?: boolean;
+    desktop?: boolean;
+    className?: string;
+    innerDivClassName?: string;
```

Comment:
are these props used? if not, I'd remove them

Reply:
Removed